templates/manager/report-view.html

```diff
@@ -0,0 +1,78 @@
+{% extends 'base.html' %}
+{% load widget_tweaks %}
+{% load filters %}
+{% block page-body %}
+<h1 class="h2 heading-underline">Email Send Statistics</h1>
+<div class="row my-5">
+  <div class="col-lg-5 col-xl-4">
+    <div class="card">
+      <div class="card-block">
+        <div class="card-title">
+          <h2 class="h4">Report Filters</h2>
+        </div>
+        <div class="card-body">
+          <form class="form-horizontal" action="" method="get">
+            <fieldset>
+              <input type="hidden" name="action" value="url_report">
+              {% for error in form.non_field_errors %}
+                <div class="alert alert-error">
+                  {{error}}
+                </div>
+              {% endfor %}
+              <div class="form-group">
+              {% for field in form %}
+                <label for="{{field.id}}" class="d-block col-form-label text-left">{{field.label}} {% if field.field.required %}*{% endif %}</label>
+                {% if field.field.required %}
+                  {{field|add_class:"form-control"|attr:"required"}}
+                {% else %}
+                  {{field|add_class:"form-control"}}
+                {% endif %}
+              {% endfor %}
+                <div class="mt-3">
+                  <button type="submit" class="btn btn-primary">Filter</button>
+                </div>
+              </div>
+            </fieldset>
+          </form>
+        </div>
+      </div>
+    </div>
+  </div>
+  <div class="col-lg-7 col-xl-8">
+    {% if not action %}
+      {% include 'manager/reports/instructions.html' %}
+    {% endif %}
+    {% if data.object_list.count %}
+      {% include templates.stats %}
+      {% include templates.data %}
+      {% if data.has_other_pages %}
+        <nav aria-label="Email instances list pagination">
+          <ul class="pagination pagination-sm">
+            {% if data.has_previous %}
+            <li class="page-item">
+              <a class="page-link" href="?{{current_url}}&page={{data.previous_page_number}}">
```

Comment:
Maybe consider renaming the variable `current_url` for clarity--e.g. maybe something like `current_query_params`?